docs/quickstart.rst

```diff
@@ -2,335 +2,227 @@
-        }
-    
-        // Request topics
-        uxrDeliveryControl delivery_control = {0};
-        delivery_control.max_samples = UXR_MAX_SAMPLES_UNLIMITED;
-        uint16_t read_data_req = uxr_buffer_request_data(&session, reliable_out, datareader_id, reliable_in, &delivery_control);
-    
-        // Read topics
-        bool connected = true;
-        while(connected && count < max_topics)
-        {
-            uint8_t read_data_status;
-            connected = uxr_run_session_until_all_status(&session, UXR_TIMEOUT_INF, &read_data_req, &read_data_status, 1);
-        }
-    
-        // Delete resources
-        uxr_delete_session(&session);
-        uxr_close_udp_transport(&transport);
-    
-        return 0;
-    }
-
-At this moment, the subscriber will receive the topics that are sending by the publisher.
-
-In order to see the messages from the DDS Global Data Space point of view, you can use *eProsima Fast RTPS* HelloWorld example running a subscriber
-(`Fast RTPS HelloWorld <http://eprosima-fast-rtps.readthedocs.io/en/latest/introduction.html#building-your-first-application>`_): ::
-
-    $ cd /usr/local/examples/C++/HelloWorldExample
-    $ sudo make && cd bin
-    $ ./HelloWorldExample subscriber
 
-Learn More
-----------
 
-To learn more about DDS and *eProsima Fast RTPS*: `eProsima Fast RTPS <http://eprosima-fast-rtps.readthedocs.io>`_
+*eProsima Micro XRCE-DDS* provides a C API which allows the creation of *eProsima Micro XRCE-DDS Clients* that can either
+publish/subscribe to topics from the DDS Global Data Space, or act as client-service applications following
+a request/reply pattern.
+
+In this section, we will guide the user through the deployment of two out-of-the-box examples.
+In the first one, an *eProsima Micro XRCE-DDS Agent*
+bridges two *eProsima Micro XRCE-DDS Clients* publishing and subscribing to the DDS world.
+The second example shows instead the deployment of a client-service application using the
+Requester and Replier entities, also put into communication via an *eProsima Micro XRCE-DDS Agent*.
+
+The section is organized as follows:
+
+- :ref:`run_agent`
+- :ref:`run_pubsub_example`
+- :ref:`run_reqrep_example`
+
+.. _run_agent:
+
+Running an Agent
+----------------
+
+First of all, install the *Agent* as explained in the :ref:`install_agent` section.
+On Linux, this would be: ::
+
+    $ git clone https://github.com/eProsima/Micro-XRCE-DDS-Agent.git
+    $ cd Micro-XRCE-DDS-Agent
+    $ mkdir build && cd build
+    $ cmake ..
+    $ make
+    $ sudo make install
+
+After having installed the *Agent* system-wide, you can launch it.
+
+For this example, the *Client*-*Agent* communication will be done through UDP, using the port :code:`2019`
+and with the XML creation mode, which is the default mode for creating entities: ::
+
+    $ cd /usr/local/bin && MicroXRCEAgent udp -p 2019
+
+.. TODO: write a comment tagging guys and asking if it was ok to remove the "-r <references-file>" part
```

Comment:
@jamoralp @pablogs9 
The original command was:
`cd /usr/local/bin && MicroXRCEAgent udp -p 2019 -r <references-file>`

Reply:
If `-r <references-file>` is explained somewhere else I think that is not necessary to have it here in the quickstart.

If you agree, accept this suggestion:

```suggestion
```